docs/src/xhtml/getstarted/index.xhtml

```diff
@@ -69,8 +69,8 @@
 				</p>
 				<figure data-ts="DoxMarkup">
 					<script type="text/html">
-						require('@tradeshift/tradeshift-ui');
-						require('@tradeshift/tradeshift-ui/ts.css');
+						import '@tradeshift/tradeshift-ui-12';
+						import '@tradeshift/tradeshift-ui-12/ts.css';
```

Comment:
Please drop the `-12`s! 
@tradeshift/tradeshift-ui-12 was a hacky solution for having 2 version of the ts-ui in one repo.

Reply:
my bad it was a copy-paste typo! thank you

Reply:
fixed